Dockerfile

```
@@ -18,6 +22,20 @@ RUN go build \
     -o /enki
 
 FROM fedora as tools-image
+COPY --from=luet /usr/bin/luet /usr/bin/luet
+ENV LUET_NOLOCK=true
+ENV TMPDIR=/tmp
+ARG TARGETARCH
+# copy both arches
+COPY luet-arm64.yaml /tmp/luet-arm64.yaml
+COPY luet-amd64.yaml /tmp/luet-amd64.yaml
+# Set the default luet config to the current build arch
+RUN mkdir -p /etc/luet/
+RUN cp /tmp/luet-${TARGETARCH}.yaml /etc/luet/luet.yaml
+## Uki artifacts, will be set under the /usr/kairos directory
+## We can install both arches, as the artifacts are named differently
+RUN luet install -y system/systemd-boot
+RUN luet install --config /tmp/luet-arm64.yaml -y system/systemd-boot
```

I think this would be problematic if started with `--platform=linux/arm64` better to always specify the config file

```suggestion
RUN luet install --config /tmp/luet-amd64.yaml -y system/systemd-boot
RUN luet install --config /tmp/luet-arm64.yaml -y system/systemd-boot
```

---

aaah good catch!